src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+        <td class="numeric">0.3%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td class="numeric">12</td>
+        <td>svg</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td class="numeric">13</td>
+        <td>eot, truetype, woff</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td class="numeric">14</td>
+        <td>opentype, woff</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td class="numeric">15</td>
+        <td>opentype</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td class="numeric">16</td>
+        <td>eot</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td class="numeric">17</td>
+        <td>opentype, svg, truetype, woff</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">18</td>
+        <td>opentype, truetype, woff, woff2</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">19</td>
+        <td>eot, truetype, woff, woff2</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">20</td>
+        <td>svg, woff</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Top 20 des combinaisons de format de polices.") }}</figcaption>
+</figure>
+
+Cet ensemble de données semble indiquer que la majorité des gens utilisent déjà WOFF2 uniquement dans leurs blocs `@font-face`. Mais cela est bien sûr trompeur, si l'on se fie à notre discussion précédente sur la domination des polices Google dans l'ensemble des données. Google Fonts utilise des méthodes de détection pour servir un fichier CSS simplifié et n'inclut que le `format()` le plus moderne. Il n'est pas surprenant que WOFF2 domine les résultats ici pour cette raison, car la prise en charge de WOFF2 par les navigateurs est assez courante depuis un certain temps déjà.
+
+Il est important de noter que ces données particulières ne soutiennent ni ne contredisent vraiment l'idée d'utiliser uniquement WOFF2 pour le moment, mais cela reste une idée tentante.
+
+## Lutter contre le texte invisible
+
+Le premier outil dont nous disposons pour lutter contre le comportement par défaut de chargement des polices web soit l'"invisibilité pendant le chargement" (aussi appelé FOIT), est `font-display`. Ajouter `font-display : swap` à votre bloc `@font-face` est un moyen facile de dire au navigateur d'afficher le texte de remplacement pendant le chargement de la police web.
```

Comment:
`"invisibilité pendant le chargement"`/`«invisibilité pendant le chargement»` (les chevrons :))

Reply:
Removed the quotes completely in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7 as not sure it makes sense to be honest.